rust/fastsim-core/src/utils.rs

```diff
@@ -294,6 +296,51 @@ pub fn interp2d(
     Ok(c)
 }
 
+lazy_static! {
+    static ref TIRE_CODE_REGEX: Regex = Regex::new(
+        r"(?i)[P|LT|ST|T]?((?:[0-9]{2,3}\.)?[0-9]+)/((?:[0-9]{1,2}\.)?[0-9]+) ?[B|D|R]?[x|\-| ]?((?:[0-9]{1,2}\.)?[0-9]+)[A|B|C|D|E|F|G|H|J|L|M|N]?"
+    ).unwrap();
+}
+
+/// Calculate tire radius (in meters) from an [ISO metric tire code](https://en.wikipedia.org/wiki/Tire_code#ISO_metric_tire_codes)
+///
+/// **Example 1:**
```

Comment:
@kylecarow , does `**Example 1:**` do anything signifiant in markdown?  If not, I'd recommend changing it to something like
```
# Examples
## Example 1 
...```

Reply:
It bolds text
\*\*example\*\* becomes **example**

I'm fine with either really
![image](https://github.com/NREL/fastsim/assets/40699307/c926d19a-bfb2-414a-a819-0adcfa7c0fcc)

![image](https://github.com/NREL/fastsim/assets/40699307/51cb3687-0b4b-4309-901b-4326bcd70d66)